components/odh-notebook-controller/controllers/notebook_controller.go

```diff
@@ -184,6 +186,12 @@ func (r *OpenshiftNotebookReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{}, err
 	}
 
+	// Call the Rolebinding reconciler
```

Comment:
can we keep this behind a devflag, like an env var which can be controlled from config level.
ex:

`if SET_ELYRA_RBAC== true`,  use this as the reference https://github.com/opendatahub-io/kubeflow/blob/6e20d786ac6035a78bac135dc1aa246b3013e3f5/components/notebook-controller/controllers/notebook_controller.go#L213
and also please upgrade the config to have this devflag set in odh-nbc manager.yaml
https://github.com/opendatahub-io/kubeflow/blob/main/components/odh-notebook-controller/config/manager/manager.yaml
example like this: https://github.com/opendatahub-io/kubeflow/blob/main/components/notebook-controller/config/manager/manager.yaml#L29

Reply:
Thank you for letting me know, for this request. 
I created the new env on this commit https://github.com/opendatahub-io/kubeflow/pull/434/commits/39773cc3b7f4172173558f8e778fbe02af4b2c7b